ee_poses:
  basket_dropoff:
    position:
      x: -0.4
      y: 0.0
      z: 0.7
    orientation:
      x: -0.105
      y: 0.994
      z: 0.0
      w: -0.035
  product_shelf_1:
    position:
      x: 1.0975
      y: 0.05
      z: 0.608
    orientation:
      x: -0.175
      y: 0.68
      z: -0.21
      w: 0.68
